.golangci.yml

```diff
@@ -34,401 +40,516 @@ linters:
     - staticcheck
     - testifylint
     - tparallel
-    - typecheck
```

Comment:
Why do you remove this linter? Is it subsumed into staticcheck as-well?

Reply:
It is enabled by default and cannot be disabled: https://golangci-lint.run/welcome/faq/#why-do-you-have-typecheck-errors
It was removed from example config file 6 months ago: https://github.com/golangci/golangci-lint/pull/5036/files

Reply:
Good to know. Thanks!